src/metabase/core.clj

```diff
@@ -105,7 +105,7 @@
     (init-status/set-progress! 0.7)
 
     ;; Now start the task runner
```

Comment:
nit: change seems to invalidate comment here, inasmuch as 'start' vs 'init' is kinda hard to differentiate